src/pytorch_lightning/utilities/apply_func.py

```diff
@@ -27,7 +27,7 @@
 
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from pytorch_lightning.utilities.imports import _compare_version, _TORCHTEXT_LEGACY
-from pytorch_lightning.utilities.warnings import rank_zero_deprecation
+from pytorch_lightning.utilities.rank_zero import rank_zero_deprecation
```

Comment:
There're two more occurrences here
```console
$ git grep "utilities.warnings import rank_zero"
src/pytorch_lightning/loggers/base.py:from pytorch_lightning.utilities.warnings import rank_zero_deprecation
src/pytorch_lightning/strategies/parallel.py:from pytorch_lightning.utilities.warnings import rank_zero_deprecation
```

Reply:
This suggestion could be ignored in favor of directly removing the deprecated methods in a separate PR